pkg/asset/internal/templates.go

```diff
@@ -560,9 +600,12 @@ spec:
         resources:
           requests:
             cpu: 150m
-            memory: 10Mi
-      - name: dnsmasq-metrics
-        image: gcr.io/google_containers/dnsmasq-metrics-amd64:1.0
+            memory: 20Mi
+        volumeMounts:
+        - name: kube-dns-config
+          mountPath: /etc/k8s/dns/dnsmasq-nanny
```

Comment:
ack. Also this is the volumeMount so that's the path in the container - so we should be fine here.